ext/node/lib.rs

```diff
@@ -41,18 +42,33 @@ pub use resolution::DEFAULT_CONDITIONS;
 
 pub trait NodeEnv {
   type P: NodePermissions;
-  // TODO(bartlomieju):
-  // type Fs: NodeFs;
+  type Fs: NodeFs;
 }
 
 pub trait NodePermissions {
   fn check_read(&mut self, path: &Path) -> Result<(), AnyError>;
 }
 
-// TODO(bartlomieju):
-// pub trait NodeFs {
-//   fn current_dir() -> Result<PathBuf, AnyError>;
-// }
+pub trait NodeFs {
```

Comment:
Seeing this used now, this means that the implementation of this trait needs to use global data/functions only, right? I think that should be ok, but it's a bit limiting.

Reply:
Did it for the first pass - if we decide that having state is needed we can change it. Doesn't seem to be necessary though